ci-operator/step-registry/telcov10n/sno-spoke/site-config/deploy/telcov10n-sno-spoke-site-config-deploy-commands.sh

```diff
@@ -119,6 +119,10 @@ function generate_site_config {
 
     generate_network_config ${baremetal_iface} ${ipi_disabled_ifaces}
 
+    if [ "${root_device}" != "" ]; then
+      ignition_config_override=$(echo -E "ignitionConfigOverride: '{\"ignition\":{\"version\":\"3.2.0\"},\"storage\":{\"disks\":[{\"device\":\"${root_device}\",\"wipeTable\":true, \"partitions\": []}]}}'")
+    fi
+
```

Comment:
This is the root cause of the problem. For the ironic agent all disks are “busy”.